Makefile

```diff
@@ -42,11 +58,25 @@ user-authenticate:
 	@read -p "User name: " uname; \
 	$(PYTHON) $(PYDIR)/manage.py drf_create_token $$uname --settings=config.settings.local
 
+# There is a race condition with how docker applies MCS
+# categories on SELinux-enabled systems. So, we need to ensure
+# the app server starts up after the web proxy.
+#
+# Users should also set:
+# setsebool -P container_connect_any on
 start-compose:
-	docker-compose up --build -d
+	@if [ $(FLAVOR) == 'redhat' ]; then \
+		docker-compose up --build -d web db queue; \
+		$(SUDO) chmod 777 $(SOCKETDIR); \
+		$(SUDO) chcon -l s0 $(SOCKETDIR); \
```

Comment:
It appears to me just from reading this that the substative difference is that on the RH box you are testing this on, SELinux is enabled. This could be the case on more than just fedora or RHEL :man_shrugging: Cursory search digs up the following unverified quote:  "SELinux is not just available for Red Hat, but it is also available out-of-the-box for a number of other distributions including Fedora, Ubuntu, Debian, Hardened Gentoo and Yellow Dog Linux."

Could test with `getenforce` or `sestatus`

Reply:
Yes, SELinux is available for more than just RH flavors. But, my scope isn't "all things running SELinux", my scope is "RH distros". Implementing support for non-RH distros is outside my scope.